src/java/org/apache/cassandra/streaming/StreamSession.java

```diff
@@ -200,6 +203,8 @@ public class StreamSession implements IEndpointStateChangeSubscriber
 
     private final TimeUUID pendingRepair;
     private final PreviewKind previewKind;
+    private final Map<String, Long> lastSeenIncomingBytesStreamed;
```

Comment:
this does not seem to be necessary anymore with all other changes in this class?